app/templates/main/volunteerProfile.html

```diff
@@ -0,0 +1,159 @@
+{% extends "base.html"%}
+{% block scripts %}
+  {{super()}}
+  <script src="/static/js/volunteerProfile.js"></script>
+{% endblock %}
+{% block styles %}
+  {{super()}}
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.css">
```

Comment:
You don't use a timepicker, do you? If not, this whole block is unnecessary.

Reply:
Yes, it is unnecessary because we are not using it. Thus, it has been removed.